template/en/default/global/user-error.html.tmpl

```diff
@@ -1401,7 +1401,7 @@
 
   [% ELSIF error == "reset_password_denied" %]
     [% title = "Reset Password Denied" %]
-    You cannot reset your password without administrative permission. 
+    You cannot reset your password without administrative permission.
```

Comment:
What's our policy on whitespace cleanup like this? Should it had been a separate PR? Since it's one line in one file it's not a blocker for me.